Scenarios/UpdateAlGoSystemFiles.md

```diff
@@ -14,15 +14,15 @@
 
    ![summary](https://github.com/microsoft/AL-Go/assets/10775043/8b87cf1e-5f39-487d-9b39-4ebf9a39706a)
 
-1. To update the AL-Go system files using the Update AL-Go System Files workflow, you need to provide a secret called GHTOKENWORKFLOW containing a Personal Access Token with permissions to modify workflows. Personal access tokens are either fine-grained personal access tokens or classic personal access tokens. AL-Go for GitHub works with both if you have enabled these tokens in organization settings:
+1. To update the AL-Go system files using the Update AL-Go System Files workflow, you need to provide a secret called GHTOKENWORKFLOW containing a Personal Access Token with permissions to modify workflows. Personal access tokens are either **fine-grained tokens** or **classic tokens**. AL-Go for GitHub works with both if you have enabled these tokens in organization settings:
```

Comment:
Should we mention somewhere that the token can't have a longer lifetime than 6-12 months? Seems like a consequence of the SFI initiatives

Reply:
I think this is only for Microsoft owned repos